_topic_map.yml

```diff
@@ -1190,9 +1190,12 @@ Topics:
 - Name: Viewing cluster logs using Kibana
   File: cluster-logging-kibana-interface
   Distros: openshift-dedicated
-- Name: Forwarding logs to third party systems
+- Name: Forwarding logs using the Log Forwarding API
   File: cluster-logging-external
   Distros: openshift-enterprise,openshift-webscale,openshift-origin
+- Name: Forwarding logs using a ConfigMap
```

Comment:
Seeing it up here makes me not like it anymore :D Maybe we should keep "Forwarding logs to third party systems" for now and describe the ConfigMap approach here, as it is whats fully supported today, and then leave "Forwarding logs using the Log Forwarding API" as a separate chapter as is. 

Also, the "to third party systems" should come before the log forwarding API.

WDYT?

Reply:
```
Forwarding logs to Third-Party System
Forwarding logs using the Log Forwarding API
```
If the first topic tells folks how to send logs to external systems, what does the Log Forward API do? <smiley-face>

Reply:
_ the API is just a CRD that the CLO takes and generates the corresponding Fluentd config._
I did not know this. (Is this true when the API goes GA? We use the Fluentd config map to send logs to Kafka, syslog, etc?)

Reply:
_When you use the Log Forwarding API, the Cluster Logging Operator creates or modifies a Fluentd ConfigMap to send logs using the Fluentd *forward* protocol. You are responsible to configure the protocol on the external log aggregator. _